controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -216,8 +216,7 @@ def test_delete_volume_still_copy_fcmaps_not_removed(self):
             self.svc.delete_volume("vol")
 
     def test_delete_volume_has_clone_fcmaps_removed(self):
-        fcmaps_as_target = self.fcmaps
-        self.svc.client.svcinfo.lsfcmap.side_effect = [Mock(as_list=fcmaps_as_target), Mock(as_list=[])]
+        self.svc.client.svcinfo.lsfcmap.side_effect = [Mock(as_list=[]), Mock(as_list=self.fcmaps_as_source)]
```

Comment:
please add a meaningful name for each mock here. e.g. fcmaps_before_x

Reply:
done